src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/sentencing/SentencingResource.kt

```diff
@@ -359,6 +359,78 @@ class SentencingResource(private val sentencingService: SentencingService) {
     sentenceSequence = sequence,
   )
 
+  @PreAuthorize("hasRole('ROLE_NOMIS_SENTENCING')")
+  @GetMapping("/prisoners/{offenderNo}/sentence-terms/booking-id/{bookingId}/sentence-sequence/{sentenceSequence}/term-sequence/{termSequence}")
+  @Operation(
```

Comment:
This url isn't consistent with the other crud operations with its use of bookingID, for the get the case Id isn't available without a call back. I will consider adding case to the trigger and come back to this